src/schemas.ts

```diff
@@ -0,0 +1,62 @@
+export default {
+  $id: 'http://graasp.org/chat/',
+  definitions: {
+    itemIdParam: {
+      type: 'object',
+      required: ['itemId'],
+      properties: {
+        itemId: { $ref: 'http://graasp.org/#/definitions/uuid' },
```

Comment:
Not sure what these refs are doing, but sometimes they have the full URLs and sometimes the short ones. What's the difference and what is this file doing?

Reply:
This is a JSON schema file that tells Fastify what shapes to expect when requests and responses are parsed / serialized (it uses `ajv` under the hood). Relative URLs are used to refer to the current definitions (for instance `#/definitions/chatMessage` refers to the `chatMessage` defined just below), while full URLs are used to reference shared schemas across the server instance (for instance the `uuid` schema is provided by the core code).